thunder/executors/torchex.py

```diff
@@ -565,6 +578,7 @@ def _squeeze_transform(a: TensorLike, /, dim: None | int | Sequence[int] = None)
 _register_implementation(ltorch.unfold, unfold, checker=_always_executable)
 _register_implementation(ltorch.unsqueeze, unsqueeze, checker=_always_executable)
 _register_implementation(ltorch.view, view, checker=_always_executable)
+_register_implementation(ltorch.empty, empty, checker=_always_executable)
```

Comment:
This needs a similar translation to the prim where the device and dtype are converted to a torch device and dtype